src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PinnedFlowSpec.groovy

```diff
@@ -110,21 +126,23 @@ class PinnedFlowSpec extends HealthCheckSpecification {
         then: "Flow is not rerouted and marked as DOWN when the first ISL is broken"
         Wrappers.wait(WAIT_OFFSET) {
             Wrappers.timedLoop(2) {
-                assert northboundV2.getFlow(flow.flowId).status == FlowState.DOWN.toString()
+                assert flow.retrieveFlowStatus().status == FlowState.DOWN
```

Comment:
Skipped, it seems like the status can be checked here because the flow details are not checked. Since only status from the Northbound API is checked, we can get only status: `flow.retrieveFlowStatus().status`